test/zippy/mzcompose.py

```diff
@@ -88,6 +94,12 @@ def workflow_default(c: Composition, parser: WorkflowArgumentParser) -> None:
     random.seed(args.seed)
 
     environment_extra = ["MZ_LOG_FILTER=warn"]
+    mz_options = [
+        "--adapter-stash-url=postgres://root@cockroach:26257?options=--search_path=adapter",
+        "--storage-stash-url=postgres://root@cockroach:26257?options=--search_path=storage",
+        "--persist-consensus-url=postgres://root@cockroach:26257?options=--search_path=consensus",
+    ]
+    persist_blob_url = "s3://minioadmin:minioadmin@persist/persist?endpoint=http://minio:9000/&region=minio"
```

Comment:
Hrm, I don't see how this is working! You never start the Cockroach container! I suspect these arguments are getting overridden by entrypoint.sh. I have a PR out to address this: https://github.com/MaterializeInc/materialize/pull/16866

Reply:
@benesch The Cockroach container is started by the `CockroachStart` Zippy action. I think the options are in effect because:
1. I see the cockroach instance in the separate CRDB container is under load
2. the Zippy test restarts that container which results in stash errors and retries in the materialize log. Ergo, this is the CRDB instance that is being used when testing.

Reply:
Ahh, ok, great. I grepped for "cockroach" in the zippy source but somehow missed this.